integration-test/helpers/mocks.js

```diff
@@ -72,5 +72,9 @@ export const constants = {
             cardNumber: true
         },
         email: true
+    },
+    iconMatchers: {
+        key: 'data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4w',
+        dax: 'data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiBo'
```

Comment:
When I tried that I think that there was a build error because the tests use `require` and the rest of the code uses `import`. I couldn't figure it out quickly and just moved on. If you or @shakyShane have suggestions, please let me know, otherwise I'd leave it be.